File: students/35274-Berke/lecture_11_task.py
class countries:
    def __init__(self, name, capital, currency, language):
        self.name = name
        self.capital = capital
        self.currency = currency
        self.language = language

    def show_info(self):
        print(f"Name: {self.name} Capital: {self.capital} Currency: {self.currency} Language: {self.language}")

country1 = countries("United Kingdom", "London", "Pound", "English")
country2 = countries("Poland", "Warsaw", "Zloty", "Polish")
country3 = countries("Turkiye", "Ankara", "Lira", "Turkish")

country1.show_info()
country2.show_info()
country3.show_info()

File: students/35274-Berke/lecture_11_task.cs
using System;

class Countries
{
    public string Name { get; set; }
    public string Capital { get; set; }
    public string Currency { get; set; }
    public string Language { get; set; }

    public Countries(string name, string capital, string currency, string language)
    {
        Name = name;
        Capital = capital;
        Currency = currency;
        Language = language;
    }

    public void ShowInfo()
    {
        Console.WriteLine($"Name: {Name} Capital: {Capital} Currency: {Currency} Language: {Language}");
    }
}

class Program
{
    static void Main(string[] args)
    {
        Countries country1 = new Countries("United Kingdom", "London", "Pound", "English");
        Countries country2 = new Countries("Poland", "Warsaw", "Zloty", "Polish");
        Countries country3 = new Countries("Turkiye", "Ankara", "Lira", "Turkish");

        country1.ShowInfo();
        country2.ShowInfo();
        country3.ShowInfo();
    }
}
